qbft/controller.go

```diff
@@ -62,6 +62,15 @@ func (c *Controller) ProcessMsg(msg *SignedMessage) (*SignedMessage, error) {
 		return nil, errors.Wrap(err, "invalid msg")
 	}
 
+	// isFutureMessage returns true if message height is from a future instance.
+	// It takes into consideration a special case where FirstHeight didn't start but  c.Height == FirstHeight (since we bump height on start instance)
+	var isFutureMessage = func(msgHeight Height) bool {
```

Comment:
this is only used once and inside `ProcessMsg`, I still don't see why it has to be declared as a `var` within the function. 
I'd advise to either of the following
1. take it out as a separate function (saves memory as this is declared again everytime `ProcessMsg` is called.
2. just use the actual comparison in line 83 like this - 
` (c.Height == FirstHeight && c.StoredInstances.FindInstance(c.Height) == nil) || msgHeight > c.Height`

Reply:
fixed